cmd/runner/README.md

```diff
@@ -15,11 +15,12 @@ bazel run //scripts:run_server -- --use_azure_runner
 
 ### Creating a new docker image to run locally
 
+When testing locally (e.g. without `--use_azure_runner`), you can build and tag a runner image locally and use that. To do that, run `bazel run //scripts:build_and_load_runner`
```

Comment:
This leaves ambiguous what happens if you're not using either of these options? Could you make that a bit clearer?

Reply:
GC, made this section more explicit